src/crates/shared/src/configuration.rs

```diff
@@ -172,6 +173,27 @@ pub struct Logging {
     pub stdout: bool,
 }
 
+#[derive(Debug, Clone, Deserialize)]
+pub enum LogginLevel {
+    Error,
+    Warn,
+    Info,
+    Debug,
+    Trace,
+}
```

Comment:
In previous comment I mentioned that they can be case-insensitive. Here you can add `#[serde(from = "String")]` and implement the `From<String>` trait with `match`ing.
```suggestion
#[derive(Debug, Clone, Deserialize)]
#[serde(from = "String")
pub enum LogginLevel {
    Error,
    Warn,
    Info,
    Debug,
    Trace,
}

impl From<String> for LoggingLevel {
    fn from(value: String) -> Self {
        match value.as_str().to_lowercase() {
            "error" => LoggingLevel::Error,
            "warn" => LoggingLevel::Warn,
            "info" => LoggingLevel::Info,
            "debug" => LoggingLevel::Debug,
            "trace" => LoggingLevel::Trace,
        }
    }
}
```

Reply:
sounds cool